projects/angular/storyteller/src/app/components/feed/feed.component.html

```diff
@@ -15,29 +15,18 @@
           <a [routerLink]="'/info/' + story.getAuthor()">
             <img class="userImage" src="/assets/user.png">
           </a>
-          <span class="headline">
-            <span class="author" *ngFor="let user of users">
-              <span *ngIf="user.getEmail() == story.getAuthor()">
-                {{user.getFirstName()}} {{user.getLastName()}}
-              </span>
+          <span *ngFor="let user of users" class="headline">
```

Comment:
The blocks of code still look pretty similiar: https://i.imgur.com/vjoGvcr.jpg
Could we share whole `storyListContainer`?